docs/src/testing/using-cheatcodes.md

```diff
@@ -268,3 +268,24 @@ fn mock_constructor_with_warp() {
     assert(block_timestamp == 1000, 'Wrong block timestamp');
 }
 ```
+
+### Mocking the constructor with `elect`
```

Comment:
I know it's out of scope, but do we need that many examples? I think prank one is enough.
Also the examples use the outdated (non-global) interface of cheatcodes

Reply:
+1, I don't think we need that many, I'm down to leave just prank